corehq/apps/app_manager/views/utils.py

```diff
@@ -291,16 +291,20 @@ def handle_custom_icon_edits(request, form_or_module, lang):
 
 def update_linked_app_and_notify(domain, app_id, user_id, email):
     app = get_current_app(domain, app_id)
+    subject = _("Update Status for linked app %s") % app.name
     try:
         update_linked_app(app, user_id)
     except AppLinkError as e:
         message = six.text_type(e)
     except Exception:
-        message = _("Something went wrong! There was an error. Please try again. "
-                    "If you see this error repeatedly please report it as issue.")
+        # Send an email but then crash the process
+        # so we know what the error was
+        send_html_email_async.delay(subject, email, _(
+            "Something went wrong! There was an error. Please try again. "
+            "If you see this error repeatedly please report it as issue."))
```

Comment:
nit/not sure: `as an` issue?

Reply:
Also, is it possible to include the exception message in the email itself?

Reply:
OK I'll fix the typo. In terms of putting the message there, I think we don't want that, but I will add to the text that we're already aware of the error and looking at it, since we now do get this message and we don't want people to think that unless they say something we're unaware.